PolyDeploy/Components/DataAccess/Models/APIUser.cs

```diff
@@ -24,8 +113,42 @@ public APIUser(string name, bool bypass)
 
             Name = name;
             BypassIPWhitelist = bypass;
-            APIKey = GenerateKey();
-            EncryptionKey = GenerateKey();
+
+            // Create keys and place them in the readable fields.
+            apiKey = GenerateKey();
```

Comment:
_Just a suggestion:_  
It's a good practice to avoid the usage of `this`, but then a naming convention should be used. Approximately all projects in which I was involved we used an leading `_` (underscore) for instance variables. It results in a great visible difference between local and instance variables in a class without using `this`.